processor/src/services/converters/capture-payment.converter.ts

```diff
@@ -1,16 +1,18 @@
 import { config } from '../../config/config';
 import { PaymentCaptureRequest } from '@adyen/api-library/lib/src/typings/checkout/paymentCaptureRequest';
 import { CapturePaymentRequest } from '../types/operation.type';
+import { LineItem } from '@adyen/api-library/lib/src/typings/checkout/lineItem';
 
 export class CapturePaymentConverter {
-  public convertRequest(opts: CapturePaymentRequest): PaymentCaptureRequest {
+  public convertRequest(opts: CapturePaymentRequest, lineItems: LineItem[] | undefined): PaymentCaptureRequest {
```

Comment:
Type of `LineItem[] | undefined` is exactly what the Adyen `PaymentCaptureRequest` accepts.